### PR TITLE
Add combine raw data , pipeline.py

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Hi @as2388 PTAL pending previous prs https://github.com/AfricasVoices/Project-ICRAF/pull/2 https://github.com/AfricasVoices/Project-ICRAF/pull/3 https://github.com/AfricasVoices/Project-ICRAF/pull/4.

### CODE REVIEW

**Comment by as2388** [comment]
@IsaackMwenda has merge conflicts

**Comment by IsaackMwenda** [comment]
> @IsaackMwenda has merge conflicts

@as2388  Fixed thank you.

File: pipeline.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+     print("Loading Pipeline Configuration File...")
+     with open(pipeline_configuration_file_path) as f:
+          pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
+
+     # Load phone number <-> UUID table
+     print("Loading Phone Number <-> UUID Table...")
+     with open(phone_number_uuid_table_path, "r") as f:
+          phone_number_uuid_table = PhoneNumberUuidTable.load(f)
+     
+          # Load messages
```
Comment (as2388) [inline pipeline.py:133]:
```suggestion
      # Load messages
```

File: pipeline.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+     json_output_path = args.json_output_path
+     icr_output_dir = args.icr_output_dir
+     coded_dir_path = args.coded_dir_path
+     csv_by_message_output_path = args.csv_by_message_output_path
+     csv_by_individual_output_path = args.csv_by_individual_output_path
+     production_csv_output_path = args.production_csv_output_path
+
+     message_paths = [s01e01_input_path, s01e02_input_path, s01e03_input_path, s01e04_input_path, 
+                         s01e05_input_path, s01e06_input_path]
```
Comment (as2388) [inline pipeline.py:121]:
```suggestion
                     s01e05_input_path, s01e06_input_path]
```

File: pipeline.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+     
+     # Upload to Google Drive, if requested.
+     # Note: This should happen as late as possible in order to reduce the risk of the remainder of the pipeline failing
+     # after a Drive upload has occurred. Failures could result in inconsistent outputs or outputs with no
+     # traced data log.
+     if drive_upload:
+          print("Uploading CSVs to Google Drive...")
+
+          # Fetch the Rapid Pro Token from the Google Cloud Storage URL
+          parsed_rapid_pro_token_file_url = urlparse(pipeline_configuration.drive_credentials_file_url)
```
Comment (as2388) [inline pipeline.py:169]: On ADSS we recently moved the token download to happen first, so we can test the authentication before running the rest of the pipeline. Can you do that here too?

File: pipeline.py [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+          csv_by_individual_drive_file_name = os.path.basename(csv_by_individual_drive_path)
+          drive_client_wrapper.update_or_create(csv_by_individual_output_path, csv_by_individual_drive_dir,
+                                                  target_file_name=csv_by_individual_drive_file_name,
+                                                  target_folder_is_shared_with_me=True)
+
+          production_csv_drive_dir = os.path.dirname(production_csv_drive_path)
+          production_csv_drive_file_name = os.path.basename(production_csv_drive_path)
+          drive_client_wrapper.update_or_create(production_csv_output_path, production_csv_drive_dir,
+                                                  target_file_name=production_csv_drive_file_name,
+                                                  target_folder_is_shared_with_me=True)
```
Comment (as2388) [inline pipeline.py:198]: Can you also upload the TracedData?

**Review by as2388 — APPROVED** [review]

File: pipeline.py (hunk 3 above)
Comment (IsaackMwenda) [reply pipeline.py:169]: @as2388  Let me fix this with generate analysis file pull thanks

**Pull request closed without merging**